extra/install_dep.sh

```diff
@@ -1,15 +1,23 @@
-#!/bin/sh
+#!/bin/bash
 #
```

Comment:
Could you drop this patch from this set since its not related to the memory helper?

We have 2 other PRs trying to do something similar, so just comment on them or if you do not like them do a new PR with just this patch.

https://github.com/open-iscsi/tcmu-runner/pull/589
https://github.com/open-iscsi/tcmu-runner/pull/590

Reply:
Strange coincidences!  Dropped.